README.md

```diff
@@ -1960,6 +1960,7 @@ Before hosting one, please see [shortcomings](https://en.wikipedia.org/wiki/URL_
 - [Kutt](https://kutt.it) - A modern URL shortener with support for custom domains. ([Source Code](https://github.com/thedevs-network/kutt)) `MIT` `Nodejs`
 - [liteshort](hhttps://git.ikl.sh/132ikl/liteshort) - User-friendly, actually lightweight, and configurable URL shortener. `MIT` `Python`
 - [Lstu](https://github.com/ldidry/lstu) - Lightweight URL shortener. `WTFPL` `Perl`
+- [RabiJump](https://github.com/KernelErr/RabiJump) - A simple and efficient URL shortener with web panel. `Apache-2.0` `Rust`
```

Comment:
As Docker is the only documented  installation method, please change the language tag to `Docker`